### DISCUSSION TITLE
这些消息竟然会同步到邮箱？？！

### DISCUSSION THREAD

**Discussion by HKLHaoBin** [share121/inter-knot#2634] [其他]
![Clip_2024-08-18_04-06-19](https://github.com/user-attachments/assets/48fe10c6-a979-4e3c-b410-a7bde3b91ffd)

**Comment by shxtxdy** [comment]
能关
![Screenshot_2024-08-18-04-27-41-07_149003a2d400f6adb210d7e357a3a646](https://github.com/user-attachments/assets/6e847c0a-52c9-4677-85dd-bc9b5f7a1f7b)
![Uploading Screenshot_2024-08-18-04-29-12-23_82b10d9f3584360f08225ddcebbb4f6d.jpg…]()

**Reply by HKLHaoBin** [reply]
其实我只是感叹，这开发者真是个天才，因为我正担心的是我回复的帖子没办法收到他的后续消息了，只能去看我的历史记录，就会比较麻烦，看到能同步到邮箱，我内心是感到欣喜的。

<img src="https://img-static.mihoyo.com/communityweb/upload/54f6c5f6a4bf851877158b0000e636c5.png" alt="克拉拉 给你心心" width="205" >

**Comment by shxtxdy** [comment]
![Screenshot_2024-08-18-04-29-12-23_82b10d9f3584360f08225ddcebbb4f6d](https://github.com/user-attachments/assets/a85879b5-487b-473f-9f91-371f8381ddee)

**Reply by HKLHaoBin** [reply]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by yytyfj** [comment]
GitHub是这样的

**Comment by HKLHaoBin** [comment]
其实我只是感叹，这开发者真是个天才，因为我正担心的是我回复的帖子没办法收到他的后续消息了，只能去看我的历史记录，就会比较麻烦，看到能同步到邮箱，我内心是感到欣喜的。

<img src="https://img-static.mihoyo.com/communityweb/upload/54f6c5f6a4bf851877158b0000e636c5.png" alt="克拉拉 给你心心" width="95" >
